examples/mobile/HomeApp/js/app.js

```diff
@@ -6,80 +6,170 @@ import Storage from "./clipping-storage.js";
 	var tau = window.tau,
 
 		storage = new Storage(),
-		appsList = [];
-
-	const defaultList = [
-			{
-				"appID": "vUf39tzQ3s.UIComponents",
-				"isInstalled": true,
-				"isActive": true,
-				"webClipsList": [
-					{
-						url: "webclip/apps-on-tv",
-						isSelected: false
-					},
-					{
-						url: "webclip/latest-news",
-						isSelected: false
-					}
-				]
-			},
-			{
-				"appID": "vUf39tzQ3t.UIComponents",
-				"isInstalled": true,
-				"isActive": false,
-				"webClipsList": [
-					{
-						url: "webclip/now-on-tv",
-						isSelected: false
-					},
-					{
-						url: "webclip/restaurant",
-						isSelected: true
-					}
-				]
-			},
-			{
-				"appID": "vUf39tzQ3r.UIComponents",
-				"isInstalled": false,
-				"isActive": false,
-				"webClipsList": [
-					{
-						url: "webclip/tv-remote-control",
-						isSelected: true
-					},
-					{
-						url: "webclip/weather",
-						isSelected: true
-					}
-				]
-			}
-		],
+		appsList = [],
+		socket = null;
+
+	const defaultList = [{
+			"appID": "vUf39tzQ3s.UIComponents",
+			"isInstalled": true,
+			"isActive": true,
+			"webClipsList": [
+				{
+					url: "webclip/apps-on-tv",
+					isSelected: false
+				},
+				{
+					url: "webclip/latest-news",
+					isSelected: false
+				}
+			]
+		},
+		{
+			"appID": "vUf39tzQ3t.UIComponents",
+			"isInstalled": true,
+			"isActive": false,
+			"webClipsList": [
+				{
+					url: "webclip/now-on-tv",
+					isSelected: false
+				},
+				{
+					url: "webclip/restaurant",
+					isSelected: true
+				}
+			]
+		},
+		{
+			"appID": "vUf39tzQ3r.UIComponents",
+			"isInstalled": false,
+			"isActive": false,
+			"webClipsList": [
+				{
+					url: "webclip/tv-remote-control",
+					isSelected: true
+				},
+				{
+					url: "webclip/weather",
+					isSelected: true
+				}
+			]
+		}],
 		getAppsList = new Promise((resolve) => {
-			const requestURL = "api/appslist";
+			const requestURL = "api/register";
 
 			fetch(requestURL)
 				.then((response) => response.json())
 				.then((data) => {
-					data.forEach((app) => {
-						app.webClipsList.forEach((webclip) => {
-							webclip.isSelected = false;
-						});
-					});
-					resolve(data);
+					addWSListener(data.wsPort);
+					resolve(data.apps);
 				})
 				.catch(() => {
-					// use default apps list if sth wrong
-					resolve(defaultList);
+					// use apps list from storage or default apps list if sth wrong
+					let apps = storage.readAllFromStorage(Storage.elements.APPSLIST);
+
```

Comment:
appsList already contain storage or default list in this case
how about just rejecting?